bitbot/reminder.go

```diff
@@ -0,0 +1,259 @@
+package bitbot
+
+import (
+	"fmt"
+	"github.com/whyrusleeping/hellabot"
+	log "gopkg.in/inconshreveable/log15.v2"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+//nolint gochecknoglobals
+var (
+	location   *time.Location
+	timeFormat string
+)
+
+type eventType struct {
```

Comment:
We should rename this to something that reflects its use in the `reminder` plugin.